ansible/psi/README.md

```diff
@@ -0,0 +1,36 @@
+Resources for setting up CI in PSI OpenStack
+============================================
+
+We have a "front-door-ci" project on the PSI OpenStack clouds rhos-d and rhos-01. The latter's nested KVM support and performance is good enough for our tasks runners.
+
+Access
+------
+Sign into the OpenShift web console on https://api.rhos-01.prod.psi.rdu2.redhat.com with group "redhat.com", your RedHat user name, and your Kerberos password. You need to be in the [front-door-ci group](https://rover.redhat.com/groups/group/front-door-ci) for this.
```

Comment:
what does it mean "group readhat.com"? does that refer to "Domain" field?

Reply:
Apparently, can we update that?

Reply:
Right, done.